ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -61,6 +61,7 @@ def call_child_workflow(self, workflow: Workflow, *,
         def wf(ctx: task.OrchestrationContext, inp: TInput):
             daprWfContext = DaprWorkflowContext(ctx)
             return workflow(daprWfContext, inp)
+        wf.__name__ = workflow.__name__ # copy workflow name so durabletask.worker can find the orchestrator in its registry
         return self.__obj.call_sub_orchestrator(wf, input=input, instance_id=instance_id)
```

Comment:
While hacky, this feels like the right fix given the current limitations of the durabletask-python library. Ideally we add a parameter to `call_sub_orchestrator` that allows overriding the name so that callers don't need to be aware of the internal implementation details.

@DeepanshuA thoughts?

Reply:
I'll merge this and you can consider a better solution in a future release.

Reply:
Comment line is too long. Please see Python style guide ;)